pkg/packagekit/manifest.json.in

```diff
@@ -8,7 +8,12 @@
 
     "tools": {
         "index": {
-            "label": "Software Updates"
+            "label": "Software Updates",
+            "keywords": {
+                "general": {
+                    "matches": ["package", "packagekit", "dnf", "yum", "apt-get"]
```

Comment:
OOI: Does this actually need to contain prefixes of other tags? I. e. would dropping "package" here make any practical difference?

I'd like to add "security" here.

Reply:
>  Does this actually need to contain prefixes of other tags?

No it does not, it would still match the same page.

>  I. e. would dropping "package" here make any practical difference?

The page would be still matched exactly the same, but it would show as found keyword `packagekit` and not `package` which in my option is worse keyword. I think that more users would search for `package` than  for `packagekit` and if user does not know what is `packagekit` that it may be a bit confusing. But I still want to show `packagekit` if someone types it directly.

Reply:
Yep, I thought the same, I just wanted to confirm. Keeping this open for the "+ security" bit.